test/paper_tabs_attr_test.dart

```diff
@@ -0,0 +1,54 @@
+@TestOn('browser')
+library polymer_elements.test.paper_tabs_attr_test;
+
+import 'dart:html';
+import 'package:polymer_elements/paper_tabs.dart';
+import 'package:polymer_elements/paper_tab.dart';
+import 'package:web_components/web_components.dart';
+import 'package:test/test.dart';
+import 'common.dart';
+
+/**
+ * Original tests:
+ * https://github.com/PolymerElements/paper-tabs/tree/master/test
+ */
+
+main() async {
+  await initWebComponents();
+
+  group('set the selected attribute', () {
+    PaperTabs tabs;
+
+    setUp(() {
+      tabs = fixture('basic');
+    });
+
+    test('selected value', () {
+      expect(tabs.selected, equals('bar'));
+    });
+
+    test('selected tab has iron-selected class', () {
+      expect(tabs.querySelector('[name=bar]').classes.contains('iron-selected'),
```

Comment:
just fyi there is also a contains matcher, but no need to change this unless you want to